charts/lagoon-remote/values.yaml

```diff
@@ -34,6 +34,8 @@ dockerHost:
     # If not set and create is true, a name is generated using the fullname
     # template
     name:
+    # set to true if you deploy the dockerhost to an openshift
+    isOpenshift: false
```

Comment:
we should probably make this a global variable since other services/subcharts (e.g. dioscuri) have a similar flag.

Reply:
yea, thought about this too, any idea how to do this? as here we need this as boolean, but the other charts need it as an additional entry in a yaml list, plus we cannot use golang templating in the values.yaml file :)

Reply:
since we control both the parent and subchart we can use a [global value](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values) like `.Values.global.openshift` and then do something like `coalesce .Values.openshift .Values.global.openshift` in the templates. What do you think?

Reply:
sounds good to me!